README.md

```diff
@@ -1,65 +1,62 @@
+# Obsidian Zoottelkeeper
 
+## 1. General Idea
+Following the idea of Nick Milo and the [LYT](https://www.linkingyourthinking.com/) -concept (Linking Your Thinking), an amazing way to bring structure to your files, folder and thoughts is by using **Maps of Content** (MOCs). Because even though Obsidian is generally built around the idea of being 'beyond' folder structures, you generally still need to have some sort of system to store all those juicy insights.
 
-# ZoottelKeeper: A Zookeeper for your Zettelkasten folder
-
-## What? 
-
-The main idea of Zettelkasten is the links that connect your notes. However, if you would like to organize your notes to folders, you have to set up and maintain an index Markdown file for each folder that contains all of the notes of the folder as links (wikilinks, internal md links etc.).
-Which means that if you move a file to an other folder, your have to remove its link from the index file of the source folder, and add a link to the index file of the target folder. 
-
-If you are struggling with the same problems (like me), ZoottelKeeper is your program. It watches your folder, catches the changes and updates your index files within every folder and subfolders automatically.
-
-## How does it work?
+## 2. How does it work?
 ZoottelKeeper watches the followings:
 
 - _Creation_ of files in rootFolder and any subfolders within 
 - _Deletion_ of files in rootFolder and any subfolders within 
 - _Move_ a file among rootFolder to subFolders
 - _Move_ a file among subfolders
 
-After recognizing that one of these actions happened, it creates an index file within the affected (sub)folder if it still does not exist.
-
-Its name is going to be **_Index_of_\<folder>.md** by default, its prefix (_Index_of_) is customizable in the config of the plugin.
-
-Then it writes the list of the files within that folder as wikistyled links. 
-
-WARNING! PLEASE NOTE THAT THIS PLUGIN IS CURRENTLY IN BETA, SO PLEASE TRY IT FIRST WITH A TEST VAULT AND/OR BACKUP YOUR DATA TO PREVENT DATA LOSS. 
-
-## Example
-
-Assume that we have a Zettelkasten folder with 2 subfolders `FolderA` and `FolderB`.
-
-1. If I create a note called `fileA.md` in `FolderA` then ZoottelKeeper creates an index file within `FolderA` called **_Index_of_FolderA.md** with content: 
-    - **[[fileA]]**
-    - **[[_Index_of_FolderA.md]]**
-
-2. If the file is being moved from `FolderA` to `FolderB`, **_Index_of_FolderB.md** is going to be created in `FolderB` with content
+### 2.1 Introduction
+So, the idea behind AutoMOC (and Zoottelkeeper) is to help you generate the base form of these maps automatically. It does so by indexing all the files and folders that lay in a folder, thus creating a link from the file to all it's content.

```

Comment:
Please improve the text by removing AutoMOC, because we should keep the original name.